sdk/appconfiguration/azure-spring-cloud-appconfiguration-config/src/main/java/com/microsoft/azure/spring/cloud/config/AppConfigurationPropertySourceLocator.java

```diff
@@ -212,18 +212,36 @@ private void addPropertySource(CompositePropertySource composite, ConfigStore st
             }
 
             // Setting new ETag values for Watch
-            List<ConfigurationSetting> watchKeys = new ArrayList<ConfigurationSetting>();
+            List<ConfigurationSetting> watchKeysSettings = new ArrayList<ConfigurationSetting>();
+            List<ConfigurationSetting> watchKeysFeatures = new ArrayList<ConfigurationSetting>();
```

Comment:
Why are these two named watchKeys**Settings** and watchKeys**Features**?

Reply:
This is part of the update. It is separating the loading/refreshing of feature flags from configurations, i.e. settings.